### DISCUSSION TITLE
Welcome to gort Discussions!

### DISCUSSION THREAD

**Discussion by clockworksoul** [getgort/gort#176] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by j0mar03** [comment]
hello where is the default location of development.yml? im having issue installation using docker

> gort_1      | time="2023-02-06T00:11:58Z" level=error msg="failed to generate config file hash" error="IO error\n  contains: read config.yml: is a directory" file=config.yml
gort_1      | Error: failed to generate config file hash
gort_1      |   contains: IO error
gort_1      |   contains: read config.yml: is a directory
